downstream/modules/platform/proc-controller-set-up-LDAP.adoc

```diff
@@ -39,31 +43,31 @@ However, you can install it from the `openldap-clients` package.
 . From the navigation panel, select menu:Settings[] in the {ControllerName} UI.
 . Select *LDAP settings* in the list of *Authentication* options.
 +
-You do not need multiple LDAP configurations per LDAP server, but you can configure multiple LDAP servers from this page, otherwise, leave the server at *Default*.
+You do not need multiple LDAP configurations per LDAP server, but you can configure many LDAP servers from this page, otherwise, leave the server at *Default*.
 +
 The equivalent API endpoints show `AUTH_LDAP_*` repeated: `AUTH_LDAP_1_*`, `AUTH_LDAP_2_*`, `AUTH_LDAP_5_*` to denote server designations.
-. To enter or modify the LDAP server address, click btn:[Edit] and enter in the *LDAP Server URI* field using the same format as the one pre-populated in the text field.
+. To enter or change the LDAP server address, click btn:[Edit] and enter in the *LDAP Server URI* field by using the same format as the one pre-populated in the text field.
 +
 [NOTE]
 ====
 You can specify multiple LDAP servers by separating each with spaces or commas. Click the image:question_circle.png[Tooltip,12,12] icon to comply with the correct syntax and rules.
 ====
 +
 . Enter the password to use for the binding user in the *LDAP Bind Password* text field. 
-For more information on LDAP variables, see link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/red_hat_ansible_automation_platform_installation_guide/appendix-inventory-files-vars#ref-hub-variables[Ansible automation hub variables].
+For more information about LDAP variables, see link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/red_hat_ansible_automation_platform_installation_guide/appendix-inventory-files-vars#ref-hub-variables[Ansible automation hub variables].
 . Click to select a group type from the *LDAP Group Type* menu.
 +
 The LDAP group types that are supported by {ControllerName} use the underlying link:https://django-auth-ldap.readthedocs.io/en/latest/groups.html#types-of-groups[django-auth-ldap library].
 To specify the parameters for the selected group type, see Step 15.
 . The *LDAP Start TLS* is disabled by default. 
-To enable TLS when the LDAP connection is not using SSL, set the toggle to *On*.
+To enable TLS when the LDAP connection is not using SSL/TLS, set the toggle to *On*.
```

Comment:
IMB style guide says to "Avoid using forward slashes (/) and backslashes (\) in most text and in dates." but there are exceptions. 

Can this be written as SSL or TLS? If so that would be my preference but you do whatever you think best!